nix/default.nix

```diff
@@ -144,20 +142,9 @@ assert lib.assertMsg (!hidpiXWayland) "The option `hidpiXWayland` has been remov
       (lib.optionals withSystemd [systemd])
     ];
 
-    cmakeBuildType =
```

Comment:
Should be replaced with `mesonBuildType` iirc. Check Meson's setup hook for more info.